src/sentiment-analysis-api/app.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+from flask import Flask, abort, jsonify, request, url_for
+import random
+import sqlalchemy as db
+
+
+app = Flask(__name__)
+
+engine = db.create_engine('postgresql+psycopg2://HIDDENALPHABET:password@hiddenalphabet-db.ceqxyonbodui.us-east-1.rds.amazonaws.com:5432/HiddenAlphabet')
```

Comment:
Make sure to refresh and delete these credentials. 
```suggestion
engine = db.create_engine('postgresql+psycopg2://'+os.environ['PG_PASSWORD']+':'+os.environ['PG_HOST']+'@'+os.environ['PG_PORT']+':'+os.environ['PG_PORT']+'/HiddenAlphabet')
```